Sources/SwiftProtobuf/Visitor.swift

```diff
@@ -445,8 +448,30 @@ public protocol Visitor {
   mutating func visitUnknown(bytes: Data) throws
 }
 
+/// Provides options for how visitor traversal should be carried out
+public struct TraversalOptions {
+
+  /// Determines if non-optional fields that are equal to their default values should be visited.
+  /// Defaults to `false`.
+  ///
+  /// ### When set to true:
+  /// - repeated fields: Will always be visited, even if they are empty.
+  /// - map fields: Will always be visited, even if they are empty.
+  /// - singular message and group fields: Will be visited only if they are set.
+  /// - singular primitive fields: Will always be visited under proto2. Under proto3 unset optional primitive fields will not be visited.
```

Comment:
This doc string likely needs an update since everything is about Editions now upstream.